fauna/errors.py

```diff
@@ -116,6 +136,13 @@ class ServiceInternalError(ServiceError):
 
 
 class ServiceTimeoutError(ServiceError):
```

Comment:
should these be named `Exception` not `Error` to be more pythonic?

Reply:
possibly. i can revisit this